src/helpers.js

```diff
@@ -177,6 +177,18 @@ let helpers = {
     } else {
       return emptyFrag().appendChild(document.createTextNode(res));
     }
+  },
+  repeat(context, params, bodies, helperContext) {
+    let {count} = params;
+    let {main} = bodies;
+    util.assert(typeof count === 'number' && count >= 0, '@repeat requires the `count` param, and it must be a number 0 or greater.');
```

Comment:
I would remove the count >= 0 assert.  It's possible the server would calculate count and pass back 0.  No need for that to throw an error.